playroom/snippets.tsx

```diff
@@ -310,6 +310,25 @@ const headerSnippets: Array<Snippet> = [
         />
         `,
     },
+    {
+        group: 'Headers',
+        name: 'Basic header layout (white)',
+        code: `
+        <HeaderLayout
+        isInverse={false}
```

Comment:
add 4 indentation spaces here (same for other snippets)

Reply:
I thought that the code had auto-indentation xDD